pkg/action/executor/k8s_controller.go

```diff
@@ -154,7 +152,19 @@ func (c *parentController) update(updatedSpec *k8sControllerSpec) error {
 		if c.ormClient == nil {
 			return fmt.Errorf("failed to execute action with nil ORMClient")
 		}
-		err = c.ormClient.Update(origControllerObj, c.obj, ownerInfo)
+		resourcePaths, err := getResourcePath(kind, updatedSpec)
+		if err != nil {
+			return fmt.Errorf("failed to execute action, Unable to get resource paths: %v", err)
+		}
+		ownerResources, err := c.ormClient.GetOwnerResourcesForSource(c.obj, ownerInfo, resourcePaths)
+		if err != nil {
+			glog.Warning(err.Error())
+			return fmt.Errorf("failed to execute action, Unable to get owner resources: %v", err)
```

Comment:
Nit: 

```go
fmt.Errorf("unable to get owner resources: %v", err)
```

Reply:
Same like the above comment, to know that this failure has occurred when trying to execute an action